specifications/active/aRMT-1.17.0.yml

```diff
@@ -0,0 +1,299 @@
+name: aRMT
```

Comment:
why do we need this new specification?

Reply:
It removes the topics that are already in the general spec. There isn’t a conflict though, so I can also remove this and just keep 1.16.0…

Reply:
Should it be `questionnaire_response` instead of `notification_thinc_it` and so on in that case?

Reply:
Well, so the generic topics, including `questionnaire_response` are now in the generic specification, so I thought there would be no need to duplicate them here.